types/sentry__webpack-plugin/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please don't disable any rules unless there's a specific need for it.

Reply:
Fixed.